dizoo/overcooked/league_demo/overcooked_selfplay_ppo_main.py

```diff
@@ -0,0 +1,107 @@
+from ding.policy import PPOPolicy
+# from ding.model import VAC
+from dizoo.overcooked.models.overcooked_vac import BaselineVAC
+from ding.utils import set_pkg_seed
+from dizoo.overcooked.envs import OvercookGameEnv
+from overcooked_demo_ppo_config import overcooked_demo_ppo_config
+
+def wrapped_overcookgame():
+    return OvercookGameEnv({})
+
+
+def main(cfg, seed=0, max_iterations=int(1e10)):
+    cfg.exp_name = 'selfplay_demo_ppo'
+    cfg = compile_config(
+        cfg,
+        BaseEnvManager,
+        PPOPolicy,
+        BaseLearner,
+        Episode1v1Collector,
+        OnevOneEvaluator,
+        NaiveReplayBuffer,
+        save_cfg=True
+    )
+    collector_env_num, evaluator_env_num = cfg.env.collector_env_num, cfg.env.evaluator_env_num
+    collector_env = BaseEnvManager(env_fn=[wrapped_overcookgame for _ in range(collector_env_num)], cfg=cfg.env.manager)
+    evaluator_env1 = BaseEnvManager(
+        env_fn=[wrapped_overcookgame for _ in range(evaluator_env_num)], cfg=cfg.env.manager
+    )
+    evaluator_env2 = BaseEnvManager(
+        env_fn=[wrapped_overcookgame for _ in range(evaluator_env_num)], cfg=cfg.env.manager
+    )
+
+    collector_env.seed(seed)
+    evaluator_env1.seed(seed, dynamic_seed=False)
+    evaluator_env2.seed(seed, dynamic_seed=False)
+    set_pkg_seed(seed, use_cuda=cfg.policy.cuda)
+
+    model1 = BaselineVAC(**cfg.policy.model)
+    policy1 = PPOPolicy(cfg.policy, model=model1)
+    model2 = BaselineVAC(**cfg.policy.model)
+    policy2 = PPOPolicy(cfg.policy, model=model2)
+    # eval_policy1 = EvalPolicy1()
+    # eval_policy2 = EvalPolicy2()
+
+    tb_logger = SummaryWriter(os.path.join('./{}/log/'.format(cfg.exp_name), 'serial'))
+    learner1 = BaseLearner(
+        cfg.policy.learn.learner, policy1.learn_mode, tb_logger, exp_name=cfg.exp_name, instance_name='learner1'
+    )
+    learner2 = BaseLearner(
+        cfg.policy.learn.learner, policy2.learn_mode, tb_logger, exp_name=cfg.exp_name, instance_name='learner2'
+    )
+    collector = Episode1v1Collector(
+        cfg.policy.collect.collector,
+        collector_env, [policy1.collect_mode, policy2.collect_mode],
+        tb_logger,
+        exp_name=cfg.exp_name
+    )
+    # collect_mode ppo use multimonial sample for selecting action
+    evaluator1_cfg = copy.deepcopy(cfg.policy.eval.evaluator)
+    evaluator1_cfg.stop_value = cfg.env.stop_value
+    evaluator1 = OnevOneEvaluator(
+        evaluator1_cfg,
+        evaluator_env1, [policy1.collect_mode, policy2.collect_mode],
+        tb_logger,
+        exp_name=cfg.exp_name,
+        instance_name='fixed_evaluator'
+    )
+    evaluator2_cfg = copy.deepcopy(cfg.policy.eval.evaluator)
+    evaluator2_cfg.stop_value = cfg.env.stop_value
+    evaluator2 = OnevOneEvaluator(
+        evaluator2_cfg,
+        evaluator_env2, [policy2.collect_mode, policy1.collect_mode],
+        tb_logger,
+        exp_name=cfg.exp_name,
+        instance_name='uniform_evaluator'
```

Comment:
Interesting environment. One small question. `evaluator1` and `evaluator2` look the same. Why one is `fixed_evaluator` while the other one is `uniform_evaluator`?

Reply:
Sorry, name mistake here. Should be ``selfplay_evaluator1`` and ``selfplay_evaluator2`` .
Thanks for pointing it out.